controls/src/main/java/com/aol/mobile/sdk/controls/view/ContentControlsView.java

```diff
@@ -500,7 +503,7 @@ public void render(@NonNull ViewModel vm) {
         isPlaying = vm.isStreamPlaying;
         thumbUrl = vm.thumbnailImageUrl;
         longitude = vm.compassLongitude;
-        if (vm.isCasting){
+        if (vm.isCasting || (accessibilityManager != null && accessibilityManager.isEnabled())) {
```

Comment:
Why do we need this?

Reply:
Controls are always shown when TalkBack is enabled